src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.